### DISCUSSION TITLE
Sky Go App won't work on OpenCore Legacy Macs with Sonoma

### DISCUSSION THREAD

**Discussion by BriggsBridget** [community/community#128433] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi All, 

I am new to this forum and not sure if this is the right channel to ask for help. 

I've been using a Mac and the SkyGo app for years. Everything was working fine until yesterday when the SkyGo app decided to do an auto-update. Once the update was completed, the app wouldn't launch at all. It just jumped once and then closed itself. 

FYI, I have 3 Macs, all running OpenCode Legacy with the latest Sonoma. All 3 Macs are experiencing the same issue, so I assume it is related to the new update.

I have read many articles and forums online since yesterday, but I haven't found a concrete solution. I also contacted SkyGo, but received no reply from them.

Can someone look into this and help me? Many Mac users are facing the same issue.

https://github.com/community/community/assets/172630978/571e91d7-ff6d-47b0-80b8-6aaac512c023

**Comment by marcoplus** [comment]
t also happens to me with the patch installed and on other Macs. I tried to reset the Nvram but I can't with my iMac with the Opencore patch. skygo opens and closes as in your video...

help us please... ....

**Reply by marcoplus** [reply]
old Mac

**Comment by cpfc22** [comment]
I'm also having the same issue with an intel iMac running open core legacy patcher. Sky just bounces in the dock and then stops with no opening of the application. I have also tried removing all files and anything linked to it and just doesn't want to work. 

I have a Macbook M2 laptop with apple silicone and that runs the SkyGo app without any issues. 

Also looking to get this resolved and any help would be great.

**Comment by dreamwhite** [comment]
Apparently Sky Go doesn't work when SIP is in a state different from enabled, so no solution apparently for OCLP users which must have it partially disabled :/
